### DISCUSSION TITLE
Alert for scamar

### DISCUSSION THREAD

**Discussion by Shailesh111578** [blockscout/blockscout#10600] [General]
Haw to alert the scamar to sidra family member?

**Comment by ekawibisono** [comment]
Hi @Shailesh111578 , this is blockscout github, please ask the owner of the platform.

**Comment by ubaidsh690o** [comment]
Hello.
My sidra coins are stolen by someone.
How to recover??

**Comment by ekawibisono** [comment]
hi @ubaidsh690o This is Blockscout github, please ask the Sidra Coin Team.
